packages/mcp-provider-devops/src/tools/createPullRequest.ts

```diff
@@ -0,0 +1,91 @@
+import { z } from "zod";
+import { CallToolResult } from "@modelcontextprotocol/sdk/types.js";
+import { McpTool, McpToolConfig, ReleaseState, Toolset, TelemetryService } from "@salesforce/mcp-provider-api";
+import { createPullRequest } from "../createPullRequest.js";
+import { fetchWorkItemByName } from "../getWorkItems.js";
+
+const inputSchema = z.object({
+  workItemName: z.string().min(1).describe("Exact Work Item Name to create pull request."),
+  username: z.string().describe("Username of the DevOps Center org to authenticate with")
+});
+type InputArgs = z.infer<typeof inputSchema>;
+type InputArgsShape = typeof inputSchema.shape;
+type OutputArgsShape = z.ZodRawShape;
+
+export class CreatePullRequest extends McpTool<InputArgsShape, OutputArgsShape> {
+  private readonly telemetryService: TelemetryService;
+
+  constructor(telemetryService: TelemetryService) {
+    super();
+    this.telemetryService = telemetryService;
+  }
+
+  public getReleaseState(): ReleaseState {
+    return ReleaseState.GA;
+  }
+
+  public getToolsets(): Toolset[] {
+    return [Toolset.OTHER];
+  }
+
+  public getName(): string {
+    return "create_pull_request";
+  }
+
+  public getConfig(): McpToolConfig<InputArgsShape, OutputArgsShape> {
+    return {
+      title: "Create Pull Request",
+      description: `Creates a pull request in a GitHub repository. Requires a GitHub token for authentication.`,
```

Comment:
ditto generic tool name
this description doesn't even mention devops-center and will collide with GH tools.